ui/component/or-rules/src/or-rule-list.ts

```diff
@@ -286,13 +286,13 @@ export class OrRuleList extends translate(i18next)(LitElement) {
                     </div>
         
                     <div id="header-btns">
-                        <or-mwc-input ?hidden="${this._isReadonly() || !this.selectedIds || this.selectedIds.length === 0}" type="${InputType.BUTTON}" icon="content-copy" @click="${() => this._onCopyClicked()}"></or-mwc-input>
-                        <or-mwc-input ?hidden="${this._isReadonly() || !this.selectedIds || this.selectedIds.length === 0}" type="${InputType.BUTTON}" icon="delete" @click="${() => this._onDeleteClicked()}"></or-mwc-input>
+                        <or-mwc-input ?hidden="${this._isReadonly() || !this.selectedIds || this.selectedIds.length === 0}" type="${InputType.BUTTON}" icon="content-copy" @click="${() => this._onCopyClicked()}" title="Copy rule"></or-mwc-input>
+                        <or-mwc-input ?hidden="${this._isReadonly() || !this.selectedIds || this.selectedIds.length === 0}" type="${InputType.BUTTON}" icon="delete" @click="${() => this._onDeleteClicked()}" title="Delete rule"></or-mwc-input>
                         ${addTemplate}
-                        <or-mwc-input hidden type="${InputType.BUTTON}" icon="magnify" @click="${() => this._onSearchClicked()}"></or-mwc-input>
+                        <or-mwc-input hidden type="${InputType.BUTTON}" icon="magnify" @click="${() => this._onSearchClicked()}" title="Search"></or-mwc-input>
                         
                         ${getContentWithMenuTemplate(
-            html`<or-mwc-input type="${InputType.BUTTON}" icon="sort-variant"></or-mwc-input>`,
+            html`<or-mwc-input type="${InputType.BUTTON}" icon="sort-variant" title="Sort"></or-mwc-input>`,
```

Comment:
`or-mwc-input` doesn't have a `title` property but probably should be added for consistency with standard `input` element...should then use `i18next.t("sort")

Reply:
Removed title properties for now. Want to add MDC tooltip to provide the desired result, while also being in line with accessibility guidelines. Added to backlog.